run/test/specs/group_tests_add_contact.spec.ts

```diff
@@ -56,28 +58,39 @@ async function addContactToGroup(platform: SupportedPlatformsType) {
   });
   if (!addedContact && platform === 'android') {
     await device1.navigateBack(platform);
-    await device1.clickOnByAccessibilityID('Add members');
+    await device1.clickOnElementAll(new InviteContactsButton(device1));
     await device1.selectByText('Contact', userD.userName);
   }
   // Click done/apply
   await device1.clickOnByAccessibilityID('Done');
   // Click done/apply again
   await sleepFor(1000);
   await device1.clickOnElementAll(new ApplyChanges(device1));
-  // Check config message
-  await runOnlyOnIOS(platform, () =>
-    device1.waitForControlMessageToBePresent(`${userD.userName} joined the group.`)
-  );
-  await runOnlyOnAndroid(platform, () =>
-    device1.waitForControlMessageToBePresent(`You added ${userD.userName} to the group.`)
-  );
-  // Exit to conversation list
+  // Check control message
+  // "legacyGroupMemberNew": "<b>{name}</b> joined the group.",
+  const legacyGroupMemberNew = localize('legacyGroupMemberNew')
+    .withArgs({ name: userD.userName })
+    .strip()
+    .toString();
+  await device1.waitForControlMessageToBePresent(legacyGroupMemberNew as ControlMessage);
+  // await Promise.all([
+  //   device1.waitForControlMessageToBePresent(`${userD.userName} joined the group.`),
+  // device2.waitForControlMessageToBePresent(`${userD.accountID} joined the group.`),
+  // device3.waitForControlMessageToBePresent(`${userD.accountID} joined the group.`),
+  // ]);
   await device4.navigateBack(platform);
-  // Select group conversation in list
   await device4.selectByText('Conversation list item', group.userName);
-  // Check control message
+  // Check control message on device 2 and 3
+  // Check for control message on device 4 (iOS doesn't support You)
   await runOnlyOnIOS(platform, () =>
-    device4.waitForControlMessageToBePresent(`${userD.userName} joined the group.`)
+    device4.waitForControlMessageToBePresent(legacyGroupMemberNew as ControlMessage)
   );
+  // Android supports You
+  // "legacyGroupMemberYouNew": "<b>You</b> joined the group.",
+  const legacyGroupMemberYouNew = localize('legacyGroupMemberYouNew').strip().toString();
+  await runOnlyOnAndroid(platform, () =>
+    device4.waitForControlMessageToBePresent(legacyGroupMemberYouNew as ControlMessage)
```

Comment:
any way we can avoid those `as` ?

Reply:
Well I think we'd have to type all the strings in the json. I've been adding the strings to the type ControlMessage but it still isn't entirely happy..
I think there needs to be a way to get the string value from the localize function and then make sure it matches our typed string

Reply:
If you have a better suggestion, i'm open to it

Reply:
@burtonemily apparently this is fixed in later PRs?

Reply:
Here's the commit to the new branch I created that removed all the checks for strings 
https://github.com/burtonemily/session-appium/commit/a1aca60ba2395be4796f26612b5994155e7149e9
I also posted about this in Discord, where I've updated my thinking on this, heres the post for reference https://discord.com/channels/408081923835953153/870484685426425877/1293454719422955581